pkg/trie/errors.go

```diff
@@ -0,0 +1,5 @@
+package trie
```

Comment:
Think will need copyright here

Reply:
True, is weird the license check didn't complain about it. Will add it

Reply:
Done